### ISSUE TITLE
ERROR: This device is not supported. （Is the 6S device not supported?）

### ISSUE THREAD

**Issue by Venhoe** [axi0mX/ipwndfu#192]

**Comment by riptl** [comment]
README says `S8000` and `S8003` not supported yet.

**Comment by Jury76** [comment]
No, but i think iPhone 6s (n71mAP | s8003) on iOS 9.1 (13B143) is supported now.

https://ibb.co/XL0ckZB
Above is the app screen
https://ibb.co/SNTHpTW
Above is the profile description

**Comment by Jury76** [comment]
Also, [https://youtu.be/7BFp99jPm74](iphone6s Plus is now supported as well.)

**Comment by 0x41c** [comment]
This completely sucks... if checkra1n is coming out tomorrow then how come they figured out s8000-s8003 weeks ago and we have nothing?
